schema/defs-image.json

```diff
@@ -15,22 +15,12 @@
       "id": "https://opencontainers.org/schema/image/manifestDescriptor",
       "type": "object",
       "required": [
-        "mediaType",
-        "size",
-        "digest",
+        "config",
         "platform"
       ],
       "properties": {
-        "mediaType": {
-          "description": "the mediatype of the referenced object",
-          "$ref": "#definitions/mediaType"
-        },
-        "size": {
-          "description": "the size in bytes of the referenced object",
-          "$ref": "defs.json#/definitions/int64"
-        },
-        "digest": {
-          "$ref": "#definitions/digest"
+        "config": {
+          "$ref": "#content-descriptor.json"
```

Comment:
It sounds like you're talking about [this Travis error][1] from opencontainers/image-tools#51.  But that's a Go compilation error; I don't see how it relates to the JSON Schema change you're proposing here.

[1]: https://travis-ci.org/opencontainers/image-tools/jobs/170981561#L248